.gitlab-ci.yml

```diff
@@ -262,16 +262,25 @@ build:docker:
   rules:
     - if: $RUN_TESTS_STAGING == "true"
       when: never
+    - when: on_success
   variables:
     DOCKER_REPOSITORY: mendersoftware/mender-client-docker-addons
     DOCKER_DIR: extra/mender-client-docker-addons
 
 publish:image:
+  rules:
+    - if: $RUN_TESTS_STAGING == "true"
+      when: never
+    - when: on_success
```

Comment:
The fallback rule here (and below) should be [this line](https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-docker-build.yml#L95),

Reply:
Force-pushed